2-structured-data/package.json

```diff
@@ -29,6 +30,16 @@
       "email": "anassri@google.com"
     }
   ],
+  "cloud-repo-tools": {
+    "requiresKeyFile": true,
+    "requiresProjectId": true,
+    "test": {
+      "app": {
```

Comment:
I don't think so, as long as they're specified in `config.json`.

Double-checking on Jenkins to be sure; so far, everything seems fine.